tests/v2/validation/rbac/rbac.go

```diff
@@ -170,3 +181,58 @@ func deleteLabels(labels map[string]string) {
 		}
 	}
 }
+
+func convertSetting(steveAdminClient *v1.Client, globalSetting *v1.SteveAPIObject) (*v3.Setting, error) {
+	updateSetting := &v3.Setting{}
+	err := v1.ConvertToK8sType(globalSetting.JSONResp, updateSetting)
+	if err != nil {
+		return nil, err
+	}
+	return updateSetting, err
+}
+
+func listGlobalSettings(steveclient *v1.Client) (projectNames []string, err error) {
+	globalSettings, err := steveclient.SteveType("management.cattle.io.setting").List(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	settingsNameList := make([]string, len(globalSettings.Data))
+	for idx, setting := range globalSettings.Data {
+		settingsNameList[idx] = setting.Name
+	}
+	sort.Strings(settingsNameList)
+	return settingsNameList, err
+}
+
+func editGlobalSettings(steveclient *v1.Client, globalSetting *v1.SteveAPIObject, value string) (*v1.SteveAPIObject, error) {
+	updateSetting, err := convertSetting(steveclient, globalSetting)
+	if err != nil {
+		return nil, err
+	}
+
+	updateSetting.Value = value
+	updateGlobalSetting, err := steveclient.SteveType("management.cattle.io.setting").Update(globalSetting, updateSetting)
+	return updateGlobalSetting, err
+}
+
+func getClusterConfig() *ClusterConfig {
+	nodeAndRoles := []string{
+		"--etcd",
+		"--controlplane",
+		"--worker",
+	}
+	userConfig := new(provisioning.Config)
+	config.LoadConfig(provisioning.ConfigurationFileKey, userConfig)
+
+	kubernetesVersion := userConfig.RKE1KubernetesVersions[0]
+	cni := userConfig.CNIs[0]
+	nodeProviders := userConfig.NodeProviders[0]
+
+	externalNodeProvider := provisioning.ExternalNodeProviderSetup(nodeProviders)
+
+	clusterConfig := ClusterConfig{nodeRoles: nodeAndRoles, externalNodeProvider: externalNodeProvider,
+		kubernetesVersion: kubernetesVersion, cni: cni}
```

Comment:
I'm assuming this is on a new line to not crowd line 234, is this correct?

Reply:
that is correct.